src/init.cpp

```diff
@@ -1619,7 +1619,8 @@ bool AppInitMain(InitInterfaces& interfaces)
             // first suggest a reindex
             if (!fReset) {
                 bool fRet = uiInterface.ThreadSafeQuestion(
-                    strLoadError + ".\n\n" + _("Do you want to rebuild the block database now?").translated,
+                    strprintf(_("%s.\n\nDo you want to rebuild the block database now?"), strLoadError),
+                    /*strLoadError + ".\n\n" + _("Do you want to rebuild the block database now?").translated,*/
```

Comment:
:eyes:

Reply:
It seems this line is outdated already ;)

Reply:
Sorry for a mess. Fixed in the latest push 24cae76e6476f467095327ca9970ef1a784c629b.